configs/ilql_randomwalks.yml

```diff
@@ -0,0 +1,35 @@
+model:
```

Comment:
Nit: What do you think about moving `ilql_randomwalks.yml` and `ppo_randomwalks.yml` to `examples/randomwalks/configs` (which is something done similarly in the [program synthesis example](https://github.com/CarperAI/trlx/tree/main/examples/experiments/grounded_program_synthesis/config))?

Reply:
agree